sprint 7/tema 1.js

```diff
@@ -0,0 +1,12 @@
+function isNumberDivByTen(number) {
+
+    if (number % 10 === 0) {
+        return (`The number ${number} is divided by 10.`);
```

Comment:
```suggestion
        return true;
```

In cerinta problemelor se cere ca valoarea returnata de functie sa fie un boolean: true sau false.